src/api/client/cache.ts

```diff
@@ -34,6 +35,16 @@ const getVideoKeyArgs = (args: Record<string, GetVideosQueryVariables['where']>
   return `${onlyCount}:${channelId}:${categoryId}:${channelIdIn}:${createdAtGte}:${isPublic}:${idEq}:${idIn}`
 }
 
+const getChannelKeyArgs = (args: Record<string, GetChannelsConnectionQueryVariables['where']> | null) => {
```

Comment:
Why exactly do we need this?

Reply:
because when we switch from `/channels` to '/new` we will get old results.

Reply:
Hmm okay, although I don't think we need `first` here

Reply:
removed